purchase_order_product_recommendation/tests/test_recommendation.py

```diff
@@ -237,7 +237,7 @@ def test_recommendations_by_warehouse(self):
         line_prod_3 = wizard.line_ids.filtered(lambda x: x.product_id == self.prod_3)
         self.assertEqual(line_prod_3.times_delivered, 1)
         self.assertEqual(line_prod_3.units_delivered, 13)
-        self.assertEqual(line_prod_3.units_included, 9)
+        self.assertEqual(line_prod_3.units_included, 8)
```

Comment:
Tracing this change, I see it was intruduced in the module migration. I wonder why it passed then :thinking:

Reply:
It is due to a commit introduced in odoo 2 days ago: https://github.com/odoo/odoo/commit/4625f3d

Reply:
Ok. But I wonder why this was forced in the first place: https://github.com/OCA/purchase-workflow/commit/e79a33913644c0e1af79ac2104f0781352a62133#diff-34f3284cd0131dc2295261002115060ea32adfa877cfba1bf2255f76447a036dR239-R241

Reply:
Yes, maybe this bug was introduced in v13 and that change had to be done to adapt to it...

Reply:
It was solved two days ago on this commit https://github.com/odoo/odoo/commit/4625f3d

So that's why wasn't failing since now

Reply:
I mean: the test was tricked in the migration to fit the result and pass then. Look that the triggering condition is the change of a date. Could it be one of those cases that depend on the hour of the day you run the test?